parser-typechecker/unison-parser-typechecker.cabal

```diff
@@ -2,7 +2,7 @@ name:          unison-parser-typechecker
 category:      Compiler
 version:       0.1
 license:       MIT
-cabal-version: >= 1.8
+cabal-version: >= 1.10
```

Comment:
This fixes a warning

Reply:
Normally prefer to have this sort of thing in a separate PR, but no need to redo it.